packages/obonode/obojobo-chunks-code/components/line/editor-component.js

```diff
@@ -4,7 +4,7 @@ import React from 'react'
 
 const Line = props => {
 	return (
-		<span className={'text align-left'} data-indent={props.node.data.get('content').indent}>
+		<span className={'text align-left'} data-indent={props.node.data.get('content').indent} data-hanging-indent={props.node.data.get('content').hangingIndent}>
```

Comment:
Does code need a hanging indent attribute from the content? A single line of code never wraps, it extends infinitely and creates a code scrollbar instead.